packages/providers/src/provider.ts

```diff
@@ -240,6 +289,17 @@ export default class Provider {
     return getOperationsSdk(gqlClient);
   }
 
+  /**
+   * A method to get the consensus parameters for a given node URL when we don't have access to an instance of the Provider class.
+   */
+  private static async getConsensusParamsWithoutInstance(url: string) {
+    const gqlClient = new GraphQLClient(url);
+    const operations = getOperationsSdk(gqlClient);
+    const { chain } = await operations.getChain();
+    const { consensusParameters } = processGqlChain(chain);
+    return consensusParameters;
+  }
+
```

Comment:
maybe this method can be removed as:

- to the purpose of getting params the user can: 
```
const provider = await Provider.create(url);
const chain = provider.getChain();
```

Reply:
@LuizAsFight actually this method is for internal use only and cannot be accessed by users. It is in place because inside of the new `connect` method, we need to fetch the consensus parameters _before_ calling the `Provider` constructor.

Reply:
yeah I understand I just don't see the use of it after applying the other comments I mentioned

if we can keep this logic only in `Provider.getChain` it's better. the code I provided would fit the needs of this method
```
const provider = await Provider.create(url);
const chain = provider.getChain();
```

Reply:
> It is in place because inside of the new `connect` method, we need to fetch the consensus parameters _before_ calling the `Provider` constructor.

you don't need if your static **create** function looks like this:
```
  static async create({ url, options }: { url: string, options?: ProviderOptions }) {
    const provider = new Provider(url, options);
    await provider.connect();
    return provider;
  }
```